### ISSUE TITLE
can't open file due to uppercase in the filepath

### ISSUE THREAD

**Issue by l0n3ly80y** [TheAveron/Whack-an-Asteroid#8]
error:
pygame 2.5.2 (SDL 2.28.2, Python 3.10.12)
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "/home/l0n3ly/Whack-an-Asteroid/main.py", line 24, in <module>
    from src import *
  File "/home/l0n3ly/Whack-an-Asteroid/src/__init__.py", line 2, in <module>
    from .global_var import *
  File "/home/l0n3ly/Whack-an-Asteroid/src/global_var.py", line 42, in <module>
    Win_sound = pygame.mixer.Sound("Static/Sounds/Win.wav")
FileNotFoundError: No file 'Static/Sounds/Win.wav' found in working directory '/home/l0n3ly/Whack-an-Asteroid'.

TLDR: files names make errors when they start by an uppercase (contradicts the file path in the code)
maybe it's just on my machine.

**Comment by TheAveron** [comment]
Thank you, on my side the link worked even if it didn't correspond exactly, but I've corrected the error.
